src/MapsSetup.php

```diff
@@ -47,6 +47,10 @@ private function defaultSettings() {
 		if ( $GLOBALS['egMapsGMaps3ApiKey'] === '' && array_key_exists( 'egGoogleJsApiKey', $GLOBALS ) ) {
 			$GLOBALS['egMapsGMaps3ApiKey'] = $GLOBALS['egGoogleJsApiKey'];
 		}
+
+		if ( $GLOBALS['egMapsGoogleGeocodingApiKey'] === '' && array_key_exists( 'egMapsGMaps3ApiKey', $GLOBALS ) ) {
```

Comment:
Is `egMapsGoogleGeocodingApiKey` guaranteed to exist? If not, this line will cause an issue

Reply:
I see. How about replacing `$GLOBALS['egMapsGoogleGeocodingApiKey'] === ''` with `empty($GLOBALS['egMapsGoogleGeocodingApiKey'])`?

Reply:
Sure